README.md

```diff
@@ -119,7 +119,7 @@ The Indie Dev's listed below would very much appreciate some coverage, and a fol
 | [Clorde](https://apps.apple.com/ca/app/clorde/id1544909562) | $0.99 (50% off) | iOS, iPadOS | Amol Kumar | [Website](https://amolkmr.com/projects/clorde), [Twitter](https://twitter.com/amolkmr05) | Utilities |
 | [Home Widget for HomeKit](https://apps.apple.com/app/id1579036143)                                  | $5.99 (40% off)              | iOS, iPadOS, Mac Silicon | Clement Marty | [Website](https://homewidget.app/), [Twitter](https://twitter.com/HomeWidget), [Facebook](https://www.facebook.com/HomeWidgetForHomeKit), [Instagram](https://www.instagram.com/home.widget/)  | Utilities |
 | [Past for iChat](https://apps.apple.com/us/app/past-for-ichat/id1554897185) | $9.99 (33% off) | macOS | Helge Heß | [WebSite](https://zeezide.de/en/products/past/), [Developer](https://github.com/helje5) | Utilities |
-| [MacWhisper Pro](https://www.macwhisper.com) | €10 (50% off) | macOS | Jordi Bruin | [Website](https://www.macwhisper.com), [Twitter](https://www.twitter.com/jordibruin) | Utilities |
+| [MacWhisper Pro](https://www.macwhisper.com) | €10 (50% off) | macOS | Jordi Bruin | [Website]([https://www.macwhisper.com](https://goodsnooze.gumroad.com/l/macwhisper/isd)), [Twitter](https://www.twitter.com/jordibruin) | Utilities |
```

Comment:
This looks like an invalid Markdown link - looking at the preview of this change, the word 'Website' is no longer a link.  I assume it should read:

`[Website](https://goodsnooze.gumroad.com/l/macwhisper/isd)`

?

Reply:
yep, sorry about that!